website/content/docs/intro/use-cases/migrate-clouds.mdx

```diff
@@ -0,0 +1,258 @@
+  - Needs credentials to your registry
+- Docker registry
+  - Credentials to push a build to the registry
+- [AWS EKS](https://docs.aws.amazon.com/eks/latest/userguide/clusters.html) cluster
+- [GCP GKE](https://cloud.google.com/kubernetes-engine) cluster
+- [Azure AKS](https://azure.microsoft.com/en-us/products/kubernetes-service/) cluster
+
+## Migrate from AWS EKS to GCP GKE
+
+##### Step 1. Start with an existing Kubernetes cluster and application deployed
+
+##### Step 2. Configure your kubectl context to point to your new cluster you are migrating to
+
+```shell
+# Use the context of your cluster you would like to migrate to
+$ kubectl config use-context gcp-context
+```
+
+##### Step 3. Waypoint runner install into your new cluster
+
+Install a runner into your cluster you are migrating to and configure the runner
+profile with your registry credentials.
+
+```shell
+$ waypoint runner install -platform=kubernetes -server-addr=<server_addr>  -k8s-runner-image=hashicorp/waypoint:latest -id=gcp -- -label=cloud=gcp
+✓ Finished connecting to: <server_addr>
+⠧ Installing runner...
+✓ Finished connecting to: <server_addr>
+✓ Runner "gcp" installed successfully to kubernetes
+✓ Runner profile "kubernetes-gcp" created successfully.
+✓ Waypoint runner installed with Helm!
+✓ Runner "gcp" adopted successfully.
+
+$ waypoint runner profile set -env-var=DOCKER_PWD='pwd' -name=kubernetes-gcp
+```
+
+After a successful `waypoint runner install` to your cluster you will see the
+following resources. Now, the runner and any on-demand runners have cluster
+edit permissions within your cluster.
+
+```shell
+$ kubectl get po
+NAME                                READY   STATUS    RESTARTS   AGE
+waypoint-gcp-runner-0               1/1     Running   0          3d1h
+
+$ kubectl get serviceaccount
+NAME                  SECRETS   AGE
+default               1         29d
+waypoint-runner       1         14d
+waypoint-runner-odr   1         14d
+
+$ kubectl get rolebinding
+NAME                                       ROLE               AGE
+waypoint-gcp-runner-odr-rolebinding-edit   ClusterRole/edit   14d
+waypoint-gcp-runner-rolebinding            ClusterRole/edit   14d
+```
+
+##### Step 4. Write a waypoint.hcl with the following specifications, if you have not already
+
+In the waypoint.hcl, specify the runner profile that was created with the
+`waypoint runner install` command. The `runner` stanza configures a project
+for remote operations and which runner profile to use.
+
+```hcl
+project = "hashitalk-deploy-gcp"
+
+app "hello-app-gcp" {
+  runner {
+    profile = "kubernetes-gcp"
+  }
+...
+```
+
+The `build` stanza of the "app" is configured later in the Waypoint configuration.
+This build uses the docker plugin to build an image and then pushes it to a Docker
+registry.
+
+```hcl
+...
+  build {
+    use "docker" {}
+    registry {
+      use "docker" {
+        image = var.image
```

Comment:
The examples in this section never detail out how to define `var.image`, `var.username` or `var.password` so we should include those too!

Reply:
Good catch, I did forget to add those.